ui/DesignSystem/Copyable.svelte

```diff
@@ -6,48 +6,34 @@
  LICENSE file.
 -->
 <script lang="typescript">
-  import type { SvelteComponent } from "svelte";
+  import type { Position } from "ui/DesignSystem/Tooltip.svelte";
 
   import { copyToClipboard } from "ui/src/ipc";
   import * as notification from "ui/src/notification";
-  import Icon from "./Icon";
+  import { Tooltip } from "ui/DesignSystem";
 
-  export let style = "";
-  export let copyContent = "";
-  export let notificationText = "Copied to your clipboard";
-  export let iconBeforeCopy: typeof SvelteComponent | undefined =
-    Icon.CopySmall;
-  export let iconAfterCopy: typeof SvelteComponent | undefined =
-    Icon.CheckSmall;
-
-  export let styleContent: boolean = true;
-  export let showIcon: boolean = true;
+  export let style: string | undefined = undefined;
+  export let title: string | undefined = undefined;
+  export let clipboardContent: string | undefined = undefined;
+  export let tooltipPosition: Position = "top";
 
   let slotContent: HTMLElement;
-  let copyIcon = iconBeforeCopy;
-
-  let copied = false;
 
-  export const copy = (): void => {
-    if (copied) {
-      return;
-    }
+  const tooltipTitle = title ? `Copy ${title} to clipboard` : undefined;
```

Comment:
How about using “Copy to clipboard” as the default tooltip?

Reply:
2cd0f27c9370f0e7d52281591ae3489e153c2c4e